tucson-teststand/AT/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: "3.6"
+
+services:
+
+  atmcs-sim:
+    image: ts-dockerhub.lsst.org/atmcs_sim:${CYCLE}
+    container_name: atmcs-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-atmcs.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-atmcs.log
+    volumes:
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "5"
+        max-size: "10m"
+
+  atpneumatics-sim:
+    image: ts-dockerhub.lsst.org/at_pneumatics_sim:${CYCLE}
+    container_name: atpneumatics-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-atpneumatics.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-atpneumatics.log
+    volumes:
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
```

Comment:
Same comment as before. No need to for host network.

Reply:
Done.